### ISSUE TITLE
"Cannot find timezone" on import

### ISSUE THREAD

**Issue by madduck** [pimutils/khal#876]
I get this on import, and am doing as I'm told:

```
warning: Cannot find timezone `Pacific/Auckland` in .ics file, this could be a bug, please report this issue at http://github.com/pimutils/khal/.
```

That's my local timezone. And the event I imported specifies:

```
BEGIN:VTIMEZONE
TZID:New Zealand Standard Time
BEGIN:STANDARD
DTSTART:16010101T030000
TZOFFSETFROM:+1300
TZOFFSETTO:+1200
RRULE:FREQ=YEARLY;INTERVAL=1;BYDAY=1SU;BYMONTH=4
END:STANDARD
BEGIN:DAYLIGHT
DTSTART:16010101T020000
TZOFFSETFROM:+1200
TZOFFSETTO:+1300
RRULE:FREQ=YEARLY;INTERVAL=1;BYDAY=-1SU;BYMONTH=9
END:DAYLIGHT
END:VTIMEZONE
…
DTSTART;TZID=New Zealand Standard Time:20190821T100000
```

**Comment by madduck** [comment]
> Can you please check if it says "Pacific/Auckland" somewhere else in the .ics file you want to import?

Nope, it's only otherwise specified in my local environment.

**Comment by ephase** [comment]
Hi @madduck and @geier , I think I've got the awnser :  Microsoft  Windows and  Oslon time zone format. Both are the same
  
 - `New Zealand Standard Time` is a Windows timezone format
 - `Pacific/Auckland` is the Olson format

I made a little test with `ipython` :

```ipython
In [1]: from icalendar import windows_to_olson                                 

In [2]: windows_to_olson.WINDOWS_TO_OLSON['New Zealand Standard Time']         
Out[2]: 'Pacific/Auckland'
```
`cal_from_ics()` return the `cal` object processed with `icalendar` module, and by the way transform the Windows timezone format to Olson into it.  But `ics_from_list()` search the Windows into the object and can't find it into cal, then report a warning.

Maybe we can test the timezone name in split_ics and add the Olson name in `tzs` if it is a Windows one.
